jax/_src/scipy/linalg.py

```diff
@@ -320,10 +862,77 @@ def qr(a: ArrayLike,  overwrite_a: bool = False, lwork: Any = None, *, mode: Lit
 def qr(a: ArrayLike, overwrite_a: bool = False, lwork: Any = None, mode: str = "full",
        pivoting: bool = False, check_finite: bool = True) -> tuple[Array] | tuple[Array, Array]: ...
 
-@implements(scipy.linalg.qr,
-        lax_description=_no_overwrite_and_chkfinite_doc, skip_params=('overwrite_a', 'check_finite', 'lwork'))
+
 def qr(a: ArrayLike, overwrite_a: bool = False, lwork: Any = None, mode: str = "full",
        pivoting: bool = False, check_finite: bool = True) -> tuple[Array] | tuple[Array, Array]:
+  """Compute the QR decomposition of an array
+
+  JAX implementation of :func:`scipy.linalg.qr`.
+
+  The QR decomposition of a matrix `A` is given by
+
+  .. math::
+
+     A = QR
+
+  Where `Q` is an orthogonal matrix (i.e. :math:`Q^TQ=I`) and `R` is an upper-triangular
```

Comment:
Nit: in the complex case, `Q` is unitary.

Reply:
Done